core/src/main/scala/org/apache/spark/sql/execution/columnar/ExternalStoreUtils.scala

```diff
@@ -273,6 +276,56 @@ object ExternalStoreUtils {
       })
   }
```

Comment:
This method can be made equal to BaseRelation.unhandledFilters. This wont give much benefit for column table, but for row table it will skip the Spark layer filter.